.github/workflows/archive-extension-source-code.yml

```diff
@@ -0,0 +1,77 @@
+name: Archive Extension Build And Source Code
+
+on:
+  push:
+    tags:
+      - coil-extension@*
+
+jobs:
+  save-monorepo-source-code:
+    runs-on: ubuntu-latest
+   
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Step 1 - Tar files
+        run: tar --exclude='.git' -cf monorepo.tar ${{ github.workspace }}
```

Comment:
Add a `-C`
tar --exclude='.git' -cf monorepo.tar -C `${{ github.workspace }}`

I think ... try it locally

Reply:
You need a `.` at the end